clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10674,12 +10674,8 @@ static void getTripleBasedSPIRVTransOpts(Compilation &C,
                                          ArgStringList &TranslatorArgs) {
   bool IsCPU = Triple.isSPIR() &&
                Triple.getSubArch() == llvm::Triple::SPIRSubArch_x86_64;
-  // Enable NonSemanticShaderDebugInfo.200 for CPU AOT and for non-Windows
-  const bool IsWindowsMSVC =
-      Triple.isWindowsMSVCEnvironment() ||
-      C.getDefaultToolChain().getTriple().isWindowsMSVCEnvironment();
-  const bool EnableNonSemanticDebug =
-      IsCPU || (!IsWindowsMSVC && !C.getDriver().IsFPGAHWMode());
+  // Enable NonSemanticShaderDebugInfo.200 for non-FPGA targets.
```

Comment:
This shouldn't be ABI breaking right? As in an will an object file compiled before this change still work?

Reply:
> This shouldn't be ABI breaking right? As in an will an object file compiled before this change still work?

Yes, we don't deprecate OpenCL.DebugInfo.100 , our drivers still support it. Please also note, that New debug info format was already enabled for Linux ~8 months ago. Right now I'm enabling it for Windows as well.

Reply:
cool, thx